reIndex.js

```diff
@@ -0,0 +1,30 @@
+function build() {
+  const fs = require('fs');
+  let main = '<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><title>The Elements</title><link rel="stylesheet" href="/css/styles.css"><script src="/deleteput.js"></script></head><body><h1>The Elements</h1><h2>These are all the known elements.</h2><h3>These are ';
+  const top = '</h3><div id="add" class="add" onclick='
```

Comment:
A string template literal would make this much more readable and easier to work with.